### PR TITLE
Chore: Add groupings to storybook stories

### PR DESCRIPTION
**Pull request by aronshamash**
Proposal for this issue: https://github.com/newsuk/times-components/issues/600

### Current
screen shot 2018-02-08 at 16 02 42

### Proposed
screen shot 2018-02-15 at 13 01 09

### CODE REVIEW

**Comment by colincclark** [comment]
👍 not an Organism in sight

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/ba2d55993f3ad3929555a3c097b31c105fafcb12/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/ddd82658aa700d09c5a480e9e218d4d93118d986/dextrose-test/index.html

**Comment by craigbilner** [comment]
Simple -> Primatives/bricks?
Compound -> Composed/walls?
Complex -> Pages/houses?
Other -> Helpers/foundations/clouds?

Thinks `Advertisement` definitely qualifies as complex :smirk: 

Would `Brightcove` not be in the earlier category?

`Caption` is simple too?

**Comment by jimhunty** [comment]
How about `Dumb`, `Smart`, `Page` and `Tools` ?

Dumb being simple lowest level components
Smart being those that have complex logic
Page being Article and Author Profile
Tools being things like Provider, Utils etc

**Comment by aronshamash** [comment]
So even some of the compound components are not 'smart' they have just been composed of multiple simple components. 

`Tools` could work instead of `Other`

`Page`:  I think even `author-profile` and `article` are not pages yet. There is no navigation or footer.

**Comment by aronshamash** [comment]
`Caption` now moved to Simple

File: package.json [hunk 1]
```diff
@@ -100,6 +100,7 @@
   "devDependencies": {
     "@storybook/addon-actions": "3.3.11",
     "@storybook/addon-info": "3.3.11",
+    "@storybook/addon-options": "3.3.12",
```
[the remark this answers is not in the document]
Comment (aronshamash) [reply package.json]: updated

**Comment by jimhunty** [comment]
Looking up the tree I like @craigbilner suggestion of bricks, walls, houses and clouds. But if we could manage a name grouping that was somehow print/newspaper related?

**Pull request merged**